_layouts/base.html.haml

```diff
@@ -24,6 +24,7 @@ change_frequency: monthly
     %link(rel="stylesheet"){:href => relative("/website/highlight/highlight.css")}
     %link(rel="stylesheet"){:href => relative("/website/tabzilla/tabzilla.css")}
     %link(rel="stylesheet"){:href => relative("/website/optaplannerWebsite.css")}
+    %link(rel="stylesheet"){:href => "https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css"}
```

Comment:
We can't depend on external urls like this . That server bootstrapcdn might go down. It might get hacked. Our site build locally needs to work locally when demonstrating it to a customer or conference without an internet connection available.

(note we already depend on an external url, although it's jboss.org for the RedHat tabzilla but we should fix that too probably)